pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -136,6 +136,10 @@ func (nc *nodeConfig) Configure() error {
 	if err := nc.configureNetwork(); err != nil {
 		return errors.Wrap(err, "configuring node network failed")
 	}
+	// Start Windows metrics exporter service
```

Comment:
I suggest executing this only if `windows_exporter.exe` is present. That way we can merge this feature now. Removing it would only involve updating the `Dockerfile` in case we did not get to doing the process work around the dependent repos.